pipelines/geo-equipamientos/dev_run.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Documentation 
+#
+# https://drive.google.com/file/d/1U23HS_EQQwv8C-mcC-0rjAQiFFA_kJI2/view
+
+set -e
+source .env
+
+WORKING_DIR=/tmp/geo_equipamientos
+ETL_UTILS=$DEV_DIR/gobierto-etl-utils
+ETL=$DEV_DIR/gobierto-etl-esplugues
+GOBIERTO_URL=$1
+YEAR=2022
+
+# Clean working dir
+cd $ETL_UTILS; ruby operations/prepare-working-directory/run.rb $WORKING_DIR
+
+# Download data in CSV format
+table_list="atencio_social cultura ensenyament esplais_jubilats esport mercats residencies_geriatriques salut"
+
+for table in $table_list; do 
+    echo "Processing $table"
+    cd $ETL_UTILS; ruby operations/download/run.rb "https://mun.nexusgeographics.com/geoserver/esplugues/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=esplugues%3A${table}&maxFeatures=10000&outputFormat=csv" $WORKING_DIR/${table}_raw.csv
```

Comment:
@jbb231 this line is donwloading the data to WORKING_DIR

You have defined it as `/tmp/geo_equipamientos` in line 10

Reply:
yes, but I cannot see this folder in my computer, is that normal?